File: backend/WebApi/WebApi/Responses/OrderResponse.cs
namespace WebApi.Responses
{
    public class OrderResponse
    {
        public int Id { get; set; }

        public OrderProductResponse[] OrderProducts { get; set; } = null!;

        public string Name { get; set; } = null!;

        public string LastName { get; set; } = null!;

        public string PhoneNumber { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string Country { get; set; } = null!;

        public string Region { get; set; } = null!;

        public string City { get; set; } = null!;

        public string Address { get; set; } = null!;

        public string Index { get; set; } = null!;

        public DateTime CreatedAt { get; set; }

        public decimal TotalSum { get; set; }
    }
}

File: frontend/src/responses/OrderResponse.ts
import OrderProductResponse from "./OrderProductResponse";

interface OrderResponse {
  id: number;
  orderProducts: OrderProductResponse[];
  name: string;
  lastName: string;
  phoneNumber: string;
  email: string;
  country: string;
  region: string;
  city: string;
  address: string;
  index: string;
  createdAt: string;
  totalSum: number;
}

export default OrderResponse;